queries.sql

```diff
@@ -81,4 +81,59 @@ WHERE owners.full_name = 'Dean Winchester' AND animals.escape_attempts = 0;
 
 SELECT owners.full_name, COUNT(animals.name) FROM animals 
 JOIN owners ON animals.owner_id = owners.id 
-GROUP BY (owners.full_name);
\ No newline at end of file
+GROUP BY (owners.full_name);
+
+SELECT visits.date_of_visit, animals.name, vets.name FROM visits
```

Comment:
- [ ] Here, you are supposed to single out the last animal that William Thatcher saw. In this query,  you just listed animals from last date to the current date. Please fix that. You can use `DESC
  LIMIT 1;
` at the end of this query before the semi-colons